PIFMSMain/Market/Procedures/usp_GetMarketInfo.sql

```diff
@@ -0,0 +1,31 @@
+﻿/*
+This procedure gets Market info based on the Name passed.
+If Name is not passed then returns all market info.
+*/
+CREATE PROCEDURE [Markets].[usp_GetMarketInfo]
+	@Name NVARCHAR(50) = NULL
+AS
+	IF(@Name is NULL)
```

Comment:
Please remove if condition, not required. Select all should only require for master data.

Reply:
add more fields like market id, client id